releasenotes/notes/fix-context-serializable-2c9768cbe4738b73.yaml

```
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    tracing: fix ``ddtrace.context.Context`` class not being serializable by ``pickle`` for passing context across processes.
```

The release note is a bit wordy. Can we simplify it a bit? 

```suggestion
    tracing: fix manual tracing across processes, make``ddtrace.context.Context`` pickable.
```